1Password/events-api-splunk
# Use password storage service for token management

<img width="1319" alt="Screen Shot 2021-08-05 at 3 55 37 PM" src="https://user-images.githubusercontent.com/4672202/128412871-397fb1eb-fe69-47ff-b958-43583aaf2862.png">

@nostodnayr Would you be able to advise on the copy for this warning 🙂 


How to test:
- Download version 1.4.1 from the releases for your OS
- Install it and go through the setup flow
- Enable a script
- run `make build_all_binaries build_all_apps` in your terminal
- Go back to splunk and install the build created from the previous step (make sure you click the checkmark during installation)

The app will not automatically start on completion. You can either restart splunk or disable / enable the input again.

The verify we have the newest version, open the `SPLUNK_HOME/etc/apps/onepassword_events_api/local/passwords.conf` file to verify the password has been encrypted.  Also note the plain text has been removed from `SPLUNK_HOME/etc/apps/onepassword_events_api/local/events_reporting.conf`
